app/dao/service_callback_api_dao.py

```diff
@@ -92,8 +97,8 @@ def delete_service_callback_api(service_callback_api):
 
 @transactional
 @version_class(ServiceCallbackApi)
-def suspend_service_callback_api(service_callback_api, updated_by_id):
-    service_callback_api.is_suspended = True
+def suspend_unsuspend_service_callback_api(service_callback_api, updated_by_id, suspend=False):
+    service_callback_api.is_suspended = suspend
```

Comment:
Since we have a `service_callback_api_history` table, when we un-suspend callbacks we should also `null` the `suspended_at` date column. If the date is present while `is_suspended` is False it could be confusing.

Reply:
hmm, I think a False with the datetime would show when the callback went live again? What do you think?

Reply:
Yea it would show that and will be useful later down the road for automation. The `updated_at` in the history table also shows that.

![image](https://github.com/user-attachments/assets/46a60a12-f329-4dd4-94dc-64ccf12bbec1)

We should document somewhere, that `suspended_at` can also represent a time of resumption.

Reply:
added in a comment on models.py